### ISSUE TITLE
Write importer and exporter for the DICOM file format. 

### ISSUE THREAD

**Issue by a-y-khan** [SCIInstitute/SCIRun#664]
Write importer for the DICOM file using the [GDCM library](http://gdcm.sourceforge.net/wiki/index.php/Main_Page). See ITK's wrapper.

**Comment by dcwhite** [comment]
@jessdtate User examples?

**Comment by jessdtate** [comment]
We'd have to use ITK.  are we up for that?  We can use the dicoms in Seg3D data (https://github.com/CIBC-Internal/Seg3DData)

**Comment by jessdtate** [comment]
Yeah, we could use GDCM without ITK.  Seg3D uses ITK, which I think has a larger subset of supported dicom formats, too.

**Comment by dcwhite** [comment]
Possible Kitware topic.

**Comment by dcwhite** [comment]
**Triage 2026-06:** DICOM importer via GDCM — GDCM has CMake support and is actively maintained. ITK uses a larger DICOM subset. For basic DICOM reading, GDCM without ITK is viable. Medium effort to add as a Superbuild external. Priority depends on whether active users need DICOM import.
